server/routes/addPrisoner/prisonerDetailsController.ts

```diff
@@ -73,32 +88,34 @@ export default class PrisonerDetailsController {
       body('lastName', 'Enter a last name').trim().escape().isLength({ min: 1, max: 250 }),
 
       body(['day', 'month', 'year']).trim(),
-      // 'prisonerDob' not an actual form field but used to hold validation errors
-      body('prisonerDob').custom((_value, { req }) => {
-        const day = Number.parseInt(req.body.day, 10).toString().padStart(2, '0')
-        const month = Number.parseInt(req.body.month, 10).toString().padStart(2, '0')
-        const year = Number.parseInt(req.body.year, 10).toString()
-        const date = `${year}-${month}-${day}`
-
-        if (date === 'NaN-NaN-NaN') {
-          throw new Error('Enter a date of birth')
-        }
-
-        if (date.includes('NaN')) {
-          throw new Error('Enter a date of birth and include a day, month and year')
-        }
-
-        const parsedDate = parseISO(date)
-        if (!isValid(parsedDate)) {
-          throw new Error('Date of birth must be a real date')
-        }
-
-        if (isAfter(parsedDate, new Date())) {
-          throw new Error('Date of birth must be in the past')
-        }
-
-        return true
-      }),
+      // 'prisonerDob' not an actual form field but used to store combined result and any validation errors
+      body('prisonerDob')
+        .customSanitizer((_value, { req }) => {
+          const day = Number.parseInt(req.body.day, 10).toString().padStart(2, '0')
+          const month = Number.parseInt(req.body.month, 10).toString().padStart(2, '0')
+          const year = Number.parseInt(req.body.year, 10).toString()
+          return `${year}-${month}-${day}`
+        })
```

Comment:
making this a sanitiser function means that the combined `YYY-MM-DD` value is stored as a 'field' available in the request handler